pkgs/by-name/ah/ahk_x11/package.nix

```
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, crystal
+, copyDesktopItems
+, linkFarm
+, fetchgit
+
+, gtk3
+, libxkbcommon
+, xorg
+, libnotify
+, gobject-introspection # needed to build gi-crystal
+, openbox
+, xvfb-run
+, xdotool
+
+, buildDevTarget ? false # the dev version prints debug info
+} :
+
+let
+  pname = "ahk_x11";
+  version = "1.0.3-unstable-2024-03-30"; # 1.0.3 doesn't build
+
+  crystalLib = linkFarm "ahk_x11-lib" (lib.mapAttrsToList (name: value: {
+    inherit name;
+    path = fetchgit value;
+  }) (import ./shards.nix));
+
+  inherit (xorg) libXinerama libXtst libXext libXi;
+
+in crystal.buildCrystalPackage {
+
+  inherit pname version;
+  src = fetchFromGitHub {
+    owner = "phil294";
+    repo = "AHK_X11";
+    rev = "047a89722f9680b2b753dee59e74a2d05e746849"; # version;
```

version does not work here?

---

See the comment following the definition of `version`. This is an untagged version. The latest tagged version does not build.

---

Delete the confusing comment

```suggestion
    rev = "047a89722f9680b2b753dee59e74a2d05e746849";
```